serveradmin/powerdns/models.py

```diff
@@ -1,3 +1,22 @@
 from django.db import models
 
-# Create your models here.
+
+class Domain(models.Model):
+    """PowerDNS Domain
+
+    Model to access domain in the PowerDNS db.
+    """
+    TYPES = [
+        ('MASTER', 'MASTER'),
+        ('SLAVE', 'SLAVE'),
+        ('NATIVE', 'SLAVE'),
```

Comment:
There is a movement in Software Development to go away from Master/Slave wording to Primary/Secondary or similar (https://en.wikipedia.org/wiki/Master/slave_(technology)#Terminology_concerns). Is it for this purpose here also possible?

Reply:
No it is not. These are internal operation modes of PowerDNS. But theere is indeed a mistake second value in tuple with `NATIVE` should be `NATIVE`.